UPGRADING.md

```diff
@@ -1,6 +1,18 @@
 Upgrading
 =========
 
+## 6.x to 7.x
+
+### The new Exception interceptor
+
+When building using Unity 2019+, the Bugsnag SDK now uses a new method to intercept uncaught C# exceptions. This allows us access to the original exception object, meaning more accurate exception data and full support for nested/inner exceptions.
+
+This means that uncaught C# exceptions previously reported will not group automatically in the Bugsnag dashboard with exceptions reported by the new notifier. 
+
+All errors will still be reported, but you may wish to pay extra attention to new exceptions that come through once upgrading and group them accordingly.
```

Comment:
> group them accordingly.

What do you mean by this?  I don't think there's anything that can be done?

Reply:
Yeah this is not worded correctly, i suppose i thought we should mention that you can group by context or by hash, but perhaps it's not necessary to say that

Reply:
```suggestion
The impact of this will be that some errors will effectively be duplicated between older and newer versions of your app.
```